src/scenes/home/home.js

```diff
@@ -224,9 +224,9 @@ class Home extends Component {
               )}
             />
             <Route
-              path="/gala"
+              path="/benefit"
               render={() => (
-                <Gala {...authProps} />
+                <Benefit {...authProps} />
```

Comment:
As an SEO hack, let's not piss off Google's scanner, and re-map `"/gala"` to this component!